yarn-project/package.json

```diff
@@ -3,10 +3,13 @@
   "packageManager": "yarn@3.4.1",
   "private": true,
   "scripts": {
+    "prepare": "workspaces-to-typescript-project-references --tsconfigPath tsconfig.dest.json && prettier -w */tsconfig.dest.json",
     "docs": "typedoc --out docs/dist && cd docs && yarn serve",
-    "formatting": "yarn workspaces foreach run formatting",
-    "formatting:fix": "yarn workspaces foreach run formatting:fix",
-    "build": "yarn workspaces foreach run build"
+    "formatting": "yarn workspaces foreach -v run formatting",
+    "formatting:fix": "yarn workspaces foreach -v run formatting:fix",
+    "test": "yarn workspaces foreach -v run test",
+    "build": "tsc -b tsconfig.dest.json",
+    "build:dev": "tsc -b tsconfig.dest.json --watch"
```

Comment:
This doesn't look like part of the fix, but I love it. Thanks for working on it!